features/consent/src/main/java/in/projecteka/jataayu/consent/ui/fragment/GrantedConsentListFragment.kt

```diff
@@ -130,6 +132,12 @@ class GrantedFragment : BaseFragment(), AdapterView.OnItemSelectedListener,
                     }
                 }
             })
+        parentVM.refresh.observe(viewLifecycleOwner, Observer{
+            if (it) {
+                parentVM.setIsRefreshing(false)
```

Comment:
can use `it` value. Also, stopping refresh before the API has completed is not right. user will not know what is happening, you should also check whether or not progress screen should be shown during this time.